### PR TITLE
Leverage only the CVE db from the walker in responses to CRDA for now.

### PR DESCRIPTION
**Pull request by bobmcwhirter**

### CODE REVIEW

**Comment by bobmcwhirter** [comment]
bombastic s3 failure; not mine?

File: exhort/api/src/server.rs [hunk 1]
```diff
@@ -162,9 +165,26 @@ async fn analyze(state: web::Data<AppState>, request: web::Json<AnalyzeRequest>)
...
     // our best effort and not allowing soft errors to fail the process.
     for vuln_id in vuln_ids {
-        match state.v11y_client.get_vulnerability(&vuln_id).await {
+        match state.v11y_client.get_cve(&vuln_id).await {
             Ok(vulnerabilities) => {
-                response.vulnerabilities.extend(vulnerabilities.iter().cloned());
+                if vulnerabilities.status() == 200 {
+                    let cve: Result<Box<RawValue>, _> = vulnerabilities.json().await;
+                    match cve {
+                        Ok(cve) => {
```
Comment (jcrossley3) [inline exhort/api/src/server.rs:173]:
```suggestion
                    match vulnerabilities.json::<Box<RawValue>>().await {
                        Ok(cve) => {
```
I don't know if it's better, but I don't love the shadowing of `cve`. Call it `raw` or something if you prefer the `let`.

**Review by jcrossley3 — APPROVED** [review]
I needed to add the `swagger` feature of the `auth` dep in order for mine to build locally. Not sure why CI builds it fine.

File: exhort/api/src/server.rs (hunk 1 above)
Comment (bobmcwhirter) [reply exhort/api/src/server.rs:173]: Oh, I love shadowing. Fisticuffs!

Comment (jcrossley3) [reply exhort/api/src/server.rs:173]: whatevs. you asked for a review. :man_shrugging:

**Comment by jcrossley3** [comment]
> bombastic s3 failure; not mine?

passes for me locally. i can't tell from the logs why it failed.

**Comment by bobmcwhirter** [comment]
original log was...DNS related.  :shrug:

**Pull request merged**